app/task-list/page.tsx

```diff
@@ -0,0 +1,9 @@
+import TaskList from "@/components/task-list";
+
+const TaskListPage = () => {
```

Comment:
https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts